### ISSUE TITLE
Can this be installed via Phone Gap Build with a line in config.xml?

### ISSUE THREAD

**Issue by jamminjames** [RemcoB00/cordova-phonegap_youtube_player_api_android#4]
Or is Cordova the only way to do it?

**Comment by rogerburgosaps** [comment]
+1

**Comment by RemcoB00** [comment]
PhoneGap Build support requires more work than just adding a line of code to the config.xml file since the plugin needs to download the YouTube Android Player API library from the Google Developers site. I will work on PhoneGap Build support in the next several weeks.

**Comment by hassankhan** [comment]
+1 would love to see this as a Cordova plugin rather than an NPM one

**Comment by vespino** [comment]
+1 where can I find a screenshot of how the plugin works?

**Comment by waynechristian** [comment]
did you get anywhere with phonegap build? is there a way to add this manually in build? Youtube is not playing ball with embeds on android.

**Comment by jamminjames** [comment]
Yes, if you did finish the PhoneGap Build support, the config.xml line is not shown in your instructions. Did that ever get done? Thanks.
